src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -83,11 +83,11 @@ WORKDIR "/src/$container_project_directory$"
 RUN dotnet build "$project_file_name$" -c Release -o /app
```

Comment:
If publish gets its own `/app/publish` folder should build get its own `/app/build` folder for consistency?

Reply:
That's quite reasonable.